metadata-models/src/main/pegasus/com/linkedin/glossary/GlossaryTermInfo.pdl

```diff
@@ -47,9 +47,4 @@ record GlossaryTermInfo {
    * A key-value map to capture any other non-standardized properties for the glossary term
    */
   customProperties: map[string, string] = { }
-
-  /**
-   * Schema definition of the glossary term
-   */
-  rawSchema: optional string
```

Comment:
we cannot delete fields on models. this is a breaking change. Use the deprecated annotation instead!

Reply:
done